test/test_vmap.py

```diff
@@ -2975,6 +2975,17 @@ def test_trace(self, device):
         x = torch.randn(2, 3, device=device, requires_grad=True)
         self._batched_grad_test(Tensor.trace, (x,))
 
+        # TODO: put this test to where it will be appropriate
+        from functorch import grad
+
+        x = torch.randn(3, 2, 2, device=device)
+
+        def sum_grad_trace(x):
+            return grad(torch.trace)(x).sum()
+
+        output = vmap(grad(sum_grad_trace))(x)
+        self.assertEqual(output, torch.zeros_like(output))
```

Comment:
@zou3519 I added temporarily your test in `test_trace` to ensure that we decompose correctly the op. Let me know if there is a more appropriate place for such check.

Reply:
I think this is fine